docker-compose.yml.custom-ssl

```diff
@@ -0,0 +1,73 @@
+version: '3'
```

Comment:
Can you rename as docker-compose-custom-ssl.yml to be consistent with other files